package.json

```diff
@@ -6,7 +6,9 @@
   "main": "utility.js",
   "scripts": {
     "test": "ava",
-    "perftest": "node --experimental-modules ./tests/performance/deepCopy.js"
+    "perftest": "node --experimental-modules ./tests/performance/deepCopy.js",
+    "lint-fix": "eslint --ignore-path .gitignore --fix .",
```

Comment:
For the folder to look at, I had it look at the current folder. If you want me to specify a folder instead, let me know and I can make that changes.

Reply:
good enough for now